src/main/java/com/epam/aidial/cfg/domain/mapper/LimitMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.aidial.cfg.domain.mapper;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public abstract class LimitMapper {
+
+    protected <T> void setIfNotNull(Consumer<T> setter, T value) {
+        if (value != null) {
+            setter.accept(value);
+        }
+    }
+
+    protected <T, R> T getLimitOrDefault(R limit, R defaultLimit, Function<R, T> limitValueGetter) {
```

Comment:
i'd move both method in commons.utils. there are no limits specific in both. they are completely generic
and would rename getLimitOrDefault -> getValueOrDefault